src/components/ScreenWrapper/BaseScreenWrapper.js

```diff
@@ -19,6 +20,8 @@ import ONYXKEYS from '../../ONYXKEYS';
 import {withNetwork} from '../OnyxProvider';
 import {propTypes, defaultProps} from './propTypes';
 
+propTypes.keyboardAvoidingViewBehavior = PropTypes.oneOf(['padding', 'height', 'position']).isRequired;
```

Comment:
I think we should add `''` to this bc the default behavior is different than all 3 of these